dump_utils.cpp

```diff
@@ -147,5 +147,62 @@ bool isHostQuiesced()
     }
     return false;
 }
+
+void createPEL(const std::string& dumpFilePath, const std::string& dumpFileType,
+               const int dumpId, const std::string& pelSev,
+               const std::string& errIntf)
+{
+    try
+    {
+        if (dumpFilePath.empty() || dumpFileType.empty())
+        {
+            log<level::WARNING>(
+                fmt::format("User data is not sufficient to "
+                            "create a PEL message for dump delete/offload")
+                    .c_str());
+            throw std::runtime_error("Dump delete/offload PEL not created due "
+                                     "to insufficient info passed");
+        }
+        std::string pelSevDefault =
+            "xyz.openbmc_project.Logging.Entry.Level.Informational";
+        if (!pelSev.empty())
+        {
+            pelSevDefault = pelSev;
+        }
+        constexpr auto loggerObjectPath = "/xyz/openbmc_project/logging";
+        constexpr auto loggerCreateInterface =
+            "xyz.openbmc_project.Logging.Create";
+        constexpr auto loggerService = "xyz.openbmc_project.Logging";
+        constexpr auto dumpFileString = "File Name";
+        constexpr auto dumpFileTypeString = "Dump Type";
+        constexpr auto dumpIdString = "Dump ID";
+
```

Comment:
That code appears to be completely avoiding using sdbusplus.  No, that is not my preference, and I would not suggest it to be a pattern to be copied.
It's pretty much structured as C code compiled with the C++ compiler.

this is the response from Patrick on discord, so this is not the correct way to do.

Reply:
https://github.com/openbmc/sdbusplus/blob/master/example/coroutine-example.cpp

please try out this one

Reply:
As per the discussion we are doing the co-routine stuffs in 1050/master at later stage. Going with the current changes as of now